pbspro.spec.in

```diff
@@ -424,6 +425,22 @@ if [ "$1" != "1" ]; then
 	echo
 fi
 
+%posttrans %{pbs_server}
+# The %preun section of 14.x unconditially removes /etc/init.d/pbs
+# because it does not check whether the package is being removed
+# or upgraded. Make sure it exists here.
+if [ -r %{pbs_prefix}/libexec/pbs_init.d ]; then
+	install -D %{pbs_prefix}/libexec/pbs_init.d /etc/init.d/pbs
+fi
+
+%posttrans %{pbs_execution}
+# The %preun section of 14.x unconditially removes /etc/init.d/pbs
+# because it does not check whether the package is being removed
+# or upgraded. Make sure it exists here.
+if [ -r %{pbs_prefix}/libexec/pbs_init.d ]; then
+	install -D %{pbs_prefix}/libexec/pbs_init.d /etc/init.d/pbs
+fi
+
```

Comment:
Shouldn't this change also be seen in the generated spec file?

Reply:
It was already there. That's what brought them out of sync. The files are identical with the exception of the line where the version is defined. I manually ran a diff to ensure this was the case.

Reply:
OK cool, thanks